ihmc-robotics-toolkit/src/test/java/us/ihmc/robotics/MatrixMissingToolsTest.java

```diff
@@ -313,4 +313,61 @@ public void testSetSelectedMatrixDiagonals()
          }
       }
    }
+
+   @Test
+   public void testPowerCalculation()
+   {
+      Random random = new Random(1738L);
+
+      int iters = 100;
+
+      for (int i = 0; i < iters; i++)
+      {
+         int matrixSize = random.nextInt(10);
+         DMatrixRMaj matrix = RandomMatrices_DDRM.rectangle(matrixSize, matrixSize, random);
+         DMatrixRMaj matrixPowerTest = new DMatrixRMaj(matrixSize, matrixSize);
+         int powerNumber = random.nextInt(15);
+
+         MatrixMissingTools.power(matrix, powerNumber, matrixPowerTest);
+
+         DMatrixRMaj matrixCompareTemporary = new DMatrixRMaj(matrixSize, matrixSize);
+         DMatrixRMaj matrixCompareResult = new DMatrixRMaj(matrix);
+
+         if (powerNumber > 1)
+         {
+            for (int j = 0; j < powerNumber - 1; j++)
+            {
+               CommonOps_DDRM.mult(matrix, matrixCompareResult, matrixCompareTemporary);
+               matrixCompareResult.set(matrixCompareTemporary);
+            }
+         }
+         MatrixTestTools.assertMatrixEquals(matrixPowerTest, matrixCompareResult, EPSILON);
+      }
+   }
+
+   @Test
+   public void testElementWiseLessThan()
+   {
+      Random random = new Random(41584L);
+
+      int iters = 100;
+
+      for (int i = 0; i < iters; i++)
+      {
+         int rowSize = random.nextInt(5, 10);
+         int columnSize = random.nextInt(5, 10);
+
+         DMatrixRMaj matrixLess = RandomMatrices_DDRM.rectangle(rowSize, columnSize, random);
+         DMatrixRMaj matrixMore = new DMatrixRMaj(rowSize, columnSize);
+         for (int j = 0; j < rowSize; j++)
+         {
+            for (int k = 0; k < columnSize; k++)
+            {
+               matrixMore.set(j, k, matrixLess.get(j, k) + random.nextDouble(0.01, 0.05));
+            }
+         }
+
+         assertTrue(MatrixMissingTools.elementWiseLessThan(matrixLess, matrixMore));
```

Comment:
Here I could swap the current implementation of your method with the following and your test would still pass:
```
public static boolean elementWiseLessThan(DMatrixRMaj a, DMatrixRMaj b)
{
   return true;
}
```
You're on the right track for the test, it is just incomplete.